libensemble/specs.py

```diff
@@ -528,7 +530,9 @@ def persistent_uniform(_, persis_info, gen_specs, libE_info):
 
     def decorator(func):
         setattr(func, "persis_in", fields)
-        func.__doc__ = f"\n    **Persistent Input Fields:** ``{func.persis_inputs}``\n" + func.__doc__
+        if not func.__doc__:
+            func.__doc__ = ""
```

Comment:
Is there no test using `persis_in`? (Codecov says it's not covered)

Reply:
These uncovered lines refer to user functions without docstrings, something that thankfully we have very few of! I'll see what I can do